pl_examples/domain_templates/imagenet.py

```diff
@@ -70,9 +65,8 @@ def __init__(
         lr: float = 0.1,
         momentum: float = 0.9,
         weight_decay: float = 1e-4,
-        batch_size: int = 4,
-        workers: int = 2,
-        **kwargs,
+        batch_size: int = 256,
+        workers: int = 4,
     ):
         super().__init__()
         self.save_hyperparameters()
```

Comment:
should we remove some of the argument assignments below since most of them are set inside hparams already? plus down below batch_size is updated, so I think it won't get updated in hparams

Reply:
We can remove `save_hyperparemters` instead, as the CLI saves the config anyways. Having both is redundant